components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -228,6 +228,7 @@ const UpdateOrderPopUp = ({ setMenuState, contribution, setShowPopup }) => {
                         }}
                       >
                         <StyledSelect
+                          inputId="tier-amount-select"
```

Comment:
There can be multiple popups open on the same page (though I'm probably going to deprecate this behavior in https://github.com/opencollective/opencollective-frontend/pull/6137 as it doesn't play well with the PayPal button)

Reply:
Think we can prepend this with the `contribution.id` which will take care of this. 👍🏽